test_check_limits.py

```diff
@@ -0,0 +1,79 @@
+import check_limits
+
+def check_limits_test():
+#SOC	      Temperature	    Charge Rate	
+#0 to 20	  -1 to 0	        0 to 0.1	    LOW_BREACH
+#21 to 24	  0.1 to 2.25	    0.11 to 0.14	LOW_WARNING
+#24 to 75	  2.26 to 42.75   0.15 to 0.75	NORMAL
+#76 to 80	  42.76 to 44.75  0.76 to 0.79	HIGH_WARNING
+#81 to 100	45 to 50	      0.8 to 1	    HIGH_BREACH
+
+#temparature limit tests
+  assert(check_limits.battery_is_ok((-1, 70, 0.7),"ENG") is False) # low breach 

```

Comment:
how do you verify if the strings logged are of the intended language?
Addition of tests for language verification, should be added.